guide/C/ch_capgain.xml

```diff
@@ -189,9 +189,9 @@
 
       <para>A month later, you have reason to suspect that the value of your painting has increased by $10,000
         (an unrealized gain). In order to record this you transfer $10,000 from your
-        <emphasis>Accrued Gains</emphasis> income account (<emphasis>Income:Unrealized
-        Gains</emphasis>) to your asset Unrealized Gains account (<emphasis>Assets:Fixed
-        Assets:Degas:Unrealized Gain</emphasis>). Your main window will resemble this:
+        <emphasis>Unrealized Gains</emphasis> income account (<emphasis>Income:Unrealized
+        Gains</emphasis>) to your asset <emphasis>Unrealized Gains</emphasis> account (<emphasis>Assets:Fixed
+        Assets:Degas:Unrealized Gains</emphasis>). Your main window will resemble this:
```

Comment:
According to the [https://wiki.gnucash.org/wiki/Documentation_Update_Instructions#Text_conventions](https://wiki.gnucash.org/wiki/Documentation_Update_Instructions#Text_conventions),

> All accounts named must be tagged with <emphasis>

Shall I use emphasis tags for all of account names, "income account" and "asset account"?
I think that it's too much and misleading.

For example, I'd like change them like
<emphasis>Assets: Fixed Assets Degas:Unrealized Gains</emphasis> asset account,
and remove words in parentheses.

Reply:
Please don't put anything in the documentation suggesting that users should book unrealized gains. Under IAS and most GAAPs that is necessary only for certain large financial institutions; nobody needing to do so should be using GnuCash.

Reply:
@jralls 
I understand what you say. But we must rewrite almost all of the chapter in order to remove the way to record of unrealized gains. So I can't do it now.

Reply:
> According to the https://wiki.gnucash.org/wiki/Documentation_Update_Instructions#Text_conventions,
> 
> > All accounts named must be tagged with

Thanks for finding the instruction! I have rewritten the section. feel free to add missing parts from our discussions.

> Shall I use emphasis tags for all of account names, "income account" and "asset account"?
> I think that it's too much and misleading.

No, only because the sentence contains twice `Unrealized Gains` the difference should be emphased. At first I thought, you made a copy error.
 
> For example, I'd like change them like
> Assets: Fixed Assets Degas:Unrealized Gains asset account,
> and remove words in parentheses.

 That would perhaps be clearer.

Reply:
> Please don't put anything in the documentation suggesting that users should book unrealized gains. Under IAS and most GAAPs that is necessary only for certain large financial institutions; nobody needing to do so should be using GnuCash.

Not guilty, it was already there in 2.0,
But this naive example needs that warning or we have to find a better example.

Reply:
The whole chapter should be rewritten from scratch, but that's a bit much to ask of @usamimikamen-sk.

Reply:
I agree with @fellen's comment.
Unrealized gains are important *concepts* for accounting.
So I think they should be left at least for concepts because examples in this chapter describe them very well.
Of course, we need to write a note at the beginning of the chapter.

Reply:
> > For example, I'd like change them like
> > Assets: Fixed Assets Degas:Unrealized Gains asset account,
> > and remove words in parentheses.
> 
> That would perhaps be clearer.

623b11e5

Reply:
> I agree with @fellen's comment.
> Unrealized gains are important _concepts_ for accounting.
> So I think they should be left at least for concepts because examples in this chapter describe them very well.
> Of course, we need to write a note at the beginning of the chapter.

4379746483bd99287612